src/util/can-patch-native-accessors.js

```diff
@@ -1,3 +1,8 @@
 // Any code referring to this is because it has to work around this bug in
 // WebKit: https://bugs.webkit.org/show_bug.cgi?id=49739
-export default !!(Object.getOwnPropertyDescriptor(window.Node.prototype, 'parentNode') && Object.getOwnPropertyDescriptor(window.Node.prototype, 'parentNode').get);
+
+import getPropertyDescriptor from './get-property-descriptor';
+
+const nativeParentNode = getPropertyDescriptor(Element.prototype, 'innerHTML');
```

Comment:
Apparently Opera doesn't have `parentNode` on `Node.prototype` but all browsers seem to have `innerHTML` on `Element.prototype` so this check works in all browsers now.